bin/stacks/analytics-stack.ts

```diff
@@ -183,6 +189,18 @@ export class AnalyticsStack extends cdk.NestedStack {
       ],
     });
 
+    const hardRequestTable = new aws_rs.Table(this, 'HardRequestTable', {
+      cluster: rsCluster,
+      adminUser: creds,
+      databaseName: RS_DATABASE_NAME,
+      tableName: 'HardRequests',
+      tableColumns: [
+        { name: 'requestId', dataType: RS_DATA_TYPES.UUID, distKey: true },
+        { name: 'createdAt', dataType: RS_DATA_TYPES.TIMESTAMP },
+        { name: 'createdAtMs', dataType: RS_DATA_TYPES.TIMESTAMP_MS },
```

Comment:
Anything else we want to log here for requests?

Reply:
yeah I was thinking requestId is enough because we can join with the indicative response log, but maybe it's good to log everything just in case